presentation/src/main/java/com/kdjj/presentation/view/adapter/BindingAdapter.kt

```diff
@@ -0,0 +1,34 @@
+package com.kdjj.presentation.view.adapter
+
+import android.widget.ArrayAdapter
+import android.widget.ImageView
+import android.widget.Spinner
+import androidx.databinding.BindingAdapter
+import androidx.recyclerview.widget.ListAdapter
+import androidx.recyclerview.widget.RecyclerView
+import com.bumptech.glide.Glide
+import com.kdjj.presentation.R
+
+@Suppress("UNCHECKED_CAST")
+@BindingAdapter("app:submitList")
+fun <T, VH : RecyclerView.ViewHolder> RecyclerView.submitList(list: List<T>?) {
+    (adapter as? ListAdapter<T, VH>)?.let { it.submitList(list) }
+}
+
+@BindingAdapter("app:typeSpinnerItems")
+fun Spinner.typeSpinnerItems(items: List<String>?) {
+    items ?: return
+
+    val spinnerAdapter = ArrayAdapter(
+        context,
+        R.layout.item_editor_recipe_type,
+        items
+    )
+    adapter = spinnerAdapter
+}
+
+@BindingAdapter("app:glideSrc")
+fun ImageView.glideSrc(src: String?) {
+    src ?: return
+    Glide.with(context).load(src).into(this)
```

Comment:
체인을 이용할때 한칸 씩 띄어주는게 코틀린 컨벤션으로 알고 있습니다!

```suggestion
    Glide.with(context)
    .load(src)
    .into(this)
```

Reply:
확인했습니다!